### PR TITLE
ian UID2-1417 added refresh endpoint for client side keypairs

### PR DESCRIPTION
**Pull request by Ian-Nara**
Currently have only tested using local mock storage and postman.

### CODE REVIEW

File: src/main/java/com/uid2/core/vertx/CoreVerticle.java [hunk 1]
```diff
@@ -275,6 +278,20 @@ private void handleClientRefresh(RoutingContext rc) {
         }
     }
 
+    private void handleClientSideKeypairRefresh(RoutingContext rc) {
+        try {
+            OperatorInfo info = OperatorInfo.getOperatorInfo(rc);
+            if (info.getOperatorType() != OperatorType.PUBLIC) {
```
Comment (Ian-Nara) [inline src/main/java/com/uid2/core/vertx/CoreVerticle.java:284]: Are all public operators of type trusted? Or should we check the protocol here as well.

**Comment by sunnywu** [comment]
If you haven't tried running uid2 locally End-to-end with multiple services, try to follow the instruction the Option D [here](https://atlassian.thetradedesk.com/confluence/display/UID2/UID2+CI+Dev+Process#UID2CIDevProcess-OptionD-MultiRepositoryDevelopment-uid2-dev-workspace). As you are going to add client side keypair data into operator soon, the E2E setup will be very helpful to do testing

File: src/main/java/com/uid2/core/vertx/CoreVerticle.java (hunk 1 above)
Comment (sunnywu) [reply src/main/java/com/uid2/core/vertx/CoreVerticle.java:284]: yeh public operator is all trusted

File: src/test/resources/com.uid2.core/model/test-config.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "aws_access_key_id": "no access key needed for test",
     "aws_secret_access_key": "no secret key needed for test",
     "clients_metadata_path": "clients/metadata.json",
+    "client_side_keypairs_metadata_path": "client_side_keypairs/metadata.json",
```
Comment (sunnywu) [inline src/test/resources/com.uid2.core/model/test-config.json:11]: can you add any integration/unit test similar to TestSiteSpecificMetadataPath.java? 

1. pretend it's a public operator and it should get a metadata json response successfully
2. pretend it's a private operator and it should get a HTTP 400 error

**Review by mcollins-ttd — APPROVED** [review]

File: src/test/resources/com.uid2.core/model/test-config.json (hunk 2 above)
Comment (Ian-Nara) [reply src/test/resources/com.uid2.core/model/test-config.json:11]: added TestMetadataPath.java

Comment (sunnywu) [reply src/test/resources/com.uid2.core/model/test-config.json:11]: could you rename TestMetadataPath.java to something more specific like TestClientSideKeyPairMetaDataPath?

**Comment by sunnywu** [comment]
Looks ok to me thanks! Please resolve the branch conflicts but don't merge it to master yet. Still need the keypair change merged into admin and deployed first before we can merge to master for core here. 

If there's a bit of waiting. i would suggest to rename the branch cstg so we have cstg branches for each of admin/operator/core. and we can keep them there until we are ok to merge all of these to master.

**Review by sunnywu — APPROVED** [review]

**Pull request merged**